watch/funcs_test.go

```diff
@@ -357,6 +357,102 @@ func TestServiceWatch(t *testing.T) {
 	wg.Wait()
 }
 
+func TestServiceMultipleTagsWatch(t *testing.T) {
+	t.Parallel()
+	a := agent.NewTestAgent(t.Name(), ``)
```

Comment:
```suggestion
	a := agent.NewTestAgent(t, t.Name(), ``)
```

I just merged a change this morning that passes the `t` value into the TestAgent. This is the cause of the current test failure.

Reply:
updated master merged and pushed :)